bindings/tflite/model.c

```diff
@@ -59,7 +59,8 @@ static iree_status_t _TfLiteModelInitializeModule(const void* flatbuffer_data,
       iree_vm_module_lookup_function_by_name(
           model->module, IREE_VM_FUNCTION_LINKAGE_EXPORT,
           iree_make_cstring_view("_tflite_main"), &model->exports._main),
-      "unable to find '_tflite_main' export in module");
+      "unable to find '_tflite_main' export in module, ensure the "
+      "`-iree-tflite-bindings-support` flag was set when compiling your model");
```

Comment:
Generally not a fan of leaking compiler flags into runtime binaries as if we ever change that flag name - or you came in from the python API that didn't use flags (it's `enable_tflite_bindings` there), etc - it's more confusing than not. Error messages aren't tested either and likely to bitrot the more specific they are - "module must be compiled with tflite bindings support" should be enough to google/code search/ask questions and avoids the (conceptual) dependency inversion. Basically, if you're pulling the runtime from some git revision and the compiler from another (a packaged release, potentially months+ forward/backward) you don't want anything directing you across that time gap which will change.

If we wanted to have more specific messages then the vulkan style of registered messages with short links that can be updated would be nice. `blah blah (http://blah/IREERT-0123)` kind of stuff that could redirect to the tflite configuration page or whatnot.

Reply:
Good points, though specific flags are nice and easy to search for. Updated to "unable to find '_tflite_main' export in module, module must be compiled with tflite bindings support".